network-api/networkapi/settings.py

```diff
@@ -415,12 +415,12 @@
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 
-WHITENOISE_ROOT = app('frontend')
 WHITENOISE_INDEX_FILE = True
 
 STATIC_URL = '/static/'
 STATIC_ROOT = root('staticfiles')
-STATICFILES_STORAGE = 'networkapi.utility.staticfiles.NonStrictCompressedManifestStaticFilesStorage'
```

Comment:
Do we need to stop using this overridden class to fix the issue? I think this change is why the review app isn't working.